assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -4,22 +4,57 @@ import Table from '@components/Table';
 import ReactMarkdown from 'react-markdown';
 import remarkGfm from 'remark-gfm';
 import LoadingBox from '@components/LoadingBox';
+import Modal from '@components/Modal';
 
-import { getCheckResults, getCheckDescription } from './checksUtils';
+import {
+  getCheckResults,
+  getCheckDescription,
+  getCheckRemediation,
+} from './checksUtils';
 
 import ResultsContainer from './ResultsContainer';
 import { ExecutionIcon } from './ExecutionIcon';
 import CheckResultOutline from './CheckResultOutline';
 import ExecutionHeader from './ExecutionHeader';
 
+const addHostnameToTargets = (targets, hostnames) =>
+  targets?.map((target) => {
+    const { agent_id } = target;
+
+    const { hostname } = hostnames.find(({ id }) => agent_id === id);
+    return {
+      ...target,
+      hostname,
+    };
+  });
+
+function MarkdownContent({ children }) {
+  return (
+    <ReactMarkdown className="markdown" remarkPlugins={[remarkGfm]}>
+      {children}
+    </ReactMarkdown>
+  );
+}
+
 const resultsTableConfig = {
   usePadding: false,
   columns: [
     {
       title: 'Id',
       key: 'checkID',
-      render: (checkID) => (
-        <div className="whitespace-nowrap text-jungle-green-500">{checkID}</div>
+      render: (checkID, item) => (
+        <div className="whitespace-nowrap text-jungle-green-500">
+          <span
+            className="inline-block"
+            aria-hidden="true"
+            onClick={(e) => {
+              e.stopPropagation();
+              item.onClickRemediation();
```

Comment:
You've got the `checkID` here, so you can directly open the modal with the selected ID here without the need of a function bound to the item